### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by LiconFromFramgia**
This is my 1st pull request for this repo.

### CODE REVIEW

**Comment by KaKaVip** [comment]
Please add .gradle, . idea folder to .gitignore 
ex : https://gist.github.com/iainconnor/8605514

**Comment by LiconFromFramgia** [comment]
.gitignore is updated like-

# Local configuration file (sdk path, etc)
/local.properties
# User-specific configurations
.idea/libraries/
.idea/workspace.xml
.idea/tasks.xml
.idea/.name
.idea/compiler.xml
.idea/copyright/profiles_settings.xml
.idea/encodings.xml
.idea/misc.xml
.idea/modules.xml
.idea/scopes/scope_settings.xml
.idea/vcs.xml
*.iml
.DS_Store
/build
/captures
# Gradle generated files
.gradle/
# Files for the Dalvik VM
*.dex
# Java class files
*.class
# Generated files
bin/
gen/
build/
/*/build/

# Log Files
*.log

**Comment by KaKaVip** [comment]
Please delete folder .idea, .gradle in your pull and commit --amend again

**Comment by LiconFromFramgia** [comment]
I have deleted the .gradle folder & add .idea folder into .gitignore & commit --amend again.  The .idea folder is needed in local repository because for project resource exploring.

**Comment by KaKaVip** [comment]
@LiconFromFramgia delete from commit but not delete from local
And .idea is setting only for you then so you don't add it to github ^^
Sorry but you can delete build/ from commit
Thank you !

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile 'com.android.support:appcompat-v7:23.0.0'
+    compile 'com.shamanland:facebook-like-button:0.1.8'
```
Comment (KaKaVip) [inline app/build.gradle]: Please using Facebook Android SDK, it supported with Like Button
And I think you need create App test on https://developers.facebook.com/ 
Please tell me, If you have difficulty :+1:

**Comment by KaKaVip** [comment]
@LiconFromFramgia 
All the characters please add to strings.xml 
Please change in the next pull
I'll merged this

**Pull request merged**